### ISSUE TITLE
Support for `doas`

### ISSUE THREAD

**Issue by hackdev17** [boltgolt/howdy#769]
With `doas` I was not able to run the /usr/bin/howdy even as a privileged user and it is same with `root`

```
[user@host ~]$ doas howdy test
doas (user@host) password: 
Please run this command as root:

        sudo howdy test
[user@host ~]$ su -
Password: 
[root@host ~]# howdy test
Please run this command as root:

        sudo howdy test
``` 
Suggest any configuration changes if required

More info
```
OS : Arch Linux

Package information
Name            : howdy
Version         : 2.6.1-2
Description     : Windows Hello for Linux
Architecture    : x86_64
Install Reason  : Explicitly installed
Install Script  : No
Validated By    : None

Files related to the package
[user@host ~]$ doas ls -R / | grep howdy
doas (user@host) password: 
~/Downloads/howdy
~/Downloads/howdy/dlib-models
~/Downloads/howdy/dlib-models/age-predictor
~/Downloads/howdy/dlib-models/gender-classifier
/usr/lib/security/howdy
/usr/lib/security/howdy/cli
/usr/lib/security/howdy/dlib-data
/usr/lib/security/howdy/pam-config
/usr/lib/security/howdy/recorders
/usr/share/licenses/howdy
/var/lib/pacman/local/howdy-2.6.1-2
```

**Comment by boltgolt** [comment]
That's weird because howdy checks if .geteuid() returns 0. What is your root user id?

**Comment by hackdev17** [comment]
As a normal user my id is 1000

**Comment by hackdev17** [comment]
I have tested `howdy` with `sudo` it works fine though, any file that invoke `sudo` I havent found yet, where do the `sudo` get invoked in the process `sudo howdy test`

**Comment by boltgolt** [comment]
Sudo doesn't get invoked, in a simplified way: By putting `sudo` before a command that command is executed as if the root user has executed it, so the user id of the process is 0

What does `doas id` give you?

**Comment by hackdev17** [comment]
```
[user@host ~]$ doas id
doas (user@host) password: 
uid=0(root) gid=0(root) groups=0(root)
```

**Comment by hackdev17** [comment]
The issue has now been resolved !
Following this guide

I have added the line to `/etc/pam.d/doas`
`auth sufficient pam_python.so /lib/security/howdy/pam.py`

```
[user@host ~]$ doas chmod -R 755 /lib/security/howdy
[user@host ~]$ cat /etc/pam.d/doas 
#%PAM-1.0
auth sufficient pam_python.so /lib/security/howdy/pam.py
auth            include         system-auth
account         include         system-auth
session         include         system-auth
```

```
[user@host ~]$ doas ls /
Attempting face detection
[ WARN:0@0.048] global cap_gstreamer.cpp:2784 handleMessage OpenCV | GStreamer warning: Embedded video playback halted; module source reported: Could not read from resource.
[ WARN:0@0.048] global cap_gstreamer.cpp:1679 open OpenCV | GStreamer warning: unable to start pipeline
[ WARN:0@0.048] global cap_gstreamer.cpp:1164 isPipelinePlaying OpenCV | GStreamer warning: GStreamer: pipeline have not been created
Identified face as user
bin  boot  dev  etc  home  lib  lib64  lost+found  mnt  opt  proc  root  run  sbin  srv  sys  tmp  usr  var
```

However I have to configure faces using `sudo` I can't configure using `doas` for now
with `doas` I was only able to edit the config files

**Comment by hackdev17** [comment]
Also there is no `geteuid()` in the `/usr/bin/howdy`, that was the reason I think it was not working with `doas`

```
[user@host ~]$ sudo howdy version
Howdy 2.6.1
[user@host ~]$ cat /usr/bin/howdy | grep geteuid | wc -l
0
```

**Comment by musikid** [comment]
Closed by #798.
